### PR TITLE
Signup and login pages

### PR DESCRIPTION
**Pull request by ababber**
### What this PR does (required):
- changed the style for the login page and the sign up page to align with the mock ups, added the appropriate page links for login and sign up

### Screenshots / Videos (front-end only):
https://user-images.githubusercontent.com/18492009/140877325-08b0ac3a-64c5-4828-b31d-6753b621cb60.mov

### Any information needed to test this feature (required):
- launch the app in your local environment, which will take tester to login page; to navigate to the sign up page use the 'Create an account' link or 'Sign up' button, which will take tester to the sign up page; tester can navigate back to login page by clicking the 'Login' link or 'Login' button

### Any issues with the current functionality (optional):
- the 'Become A Sitter' link navigates tester to login page because the appropriate page has not been created yet
- there were no login page mock ups in the assets for this project, so the loom video for the project walkthrough was used as a guide

### CODE REVIEW

File: client/src/pages/Login/LoginForm/LoginForm.tsx [hunk 1]
```diff
@@ -46,9 +56,9 @@ export default function Login({ handleSubmit }: Props): JSX.Element {
     >
       {({ handleSubmit, handleChange, values, touched, errors, isSubmitting }) => (
         <form onSubmit={handleSubmit} className={classes.form} noValidate>
+          <FormLabel className={classes.formLabel}>EMAIL ADDRESS</FormLabel>
```
Comment (ezeilo-su) [inline client/src/pages/Login/LoginForm/LoginForm.tsx:59]: Please avoid using CAPITALCASE for text, some screen readers interpret this text as abbreviations, you can resort to using a CSS class to transform text instead.

File: client/src/pages/Login/LoginForm/LoginForm.tsx [hunk 2]
```diff
@@ -64,10 +74,12 @@ export default function Login({ handleSubmit }: Props): JSX.Element {
             error={touched.email && Boolean(errors.email)}
             value={values.email}
             onChange={handleChange}
+            variant="outlined"
+            className={classes.textField}
           />
+          <FormLabel className={classes.formLabel}>PASSWORD</FormLabel>
```
Comment (ezeilo-su) [inline client/src/pages/Login/LoginForm/LoginForm.tsx:80]: The same thing here as above with CAPITALCASE

File: client/src/pages/SignUp/SignUpForm/SignUpForm.tsx [hunk 3]
```diff
@@ -51,9 +61,9 @@ const SignUpForm = ({ handleSubmit }: Props): JSX.Element => {
     >
       {({ handleSubmit, handleChange, values, touched, errors, isSubmitting }) => (
         <form onSubmit={handleSubmit} className={classes.form} noValidate>
+          <FormLabel className={classes.formLabel}>EMAIL ADDRESS</FormLabel>
```
Comment (ezeilo-su) [inline client/src/pages/SignUp/SignUpForm/SignUpForm.tsx:64]: Avoid the use of CAPITALCASE for text. Use CSS to transform, instead.

File: client/src/pages/SignUp/SignUpForm/SignUpForm.tsx [hunk 4]
```diff
@@ -105,12 +122,16 @@ const SignUpForm = ({ handleSubmit }: Props): JSX.Element => {
...
             onChange={handleChange}
+            variant="outlined"
+            placeholder="Create a password"
+            className={classes.textField}
           />
 
           <Box textAlign="center">
             <Button type="submit" size="large" variant="contained" color="primary" className={classes.submit}>
-              {isSubmitting ? <CircularProgress style={{ color: 'white' }} /> : 'Create'}
+              {isSubmitting ? <CircularProgress style={{ color: 'white' }} /> : 'SIGN UP'}
```
Comment (ezeilo-su) [inline client/src/pages/SignUp/SignUpForm/SignUpForm.tsx:132]: CAPITALCASE with text again. Please change it :smile:

**Review by ezeilo-su — CHANGES_REQUESTED** [review]
UI looks good. Just address those comments below.

File: client/src/themes/theme.ts [hunk 5]
```diff
@@ -10,7 +10,8 @@ export const theme = createMuiTheme({
     },
   },
   palette: {
-    primary: { main: '#3A8DFF' },
+    primary: { main: '#F14140' },
```
Comment (rajivtitus) [inline client/src/themes/theme.ts:13]: 💯

File: client/src/pages/Login/LoginForm/LoginForm.tsx (hunk 1 above)
Comment (ababber) [reply client/src/pages/Login/LoginForm/LoginForm.tsx]: Thanks for the feedback! I made changes based on your suggestion.

**Review by rajivtitus — COMMENTED** [review]
A couple of comments weren't posted from the initial review, adding them once again for you to have a look. Thanks! 😄

File: client/src/components/AuthHeader/AuthHeader.tsx [hunk 6]
```diff
@@ -1,27 +1,46 @@
...
+          <Grid container justify="space-between">
+            <Grid item>
+              <img src={logo} alt="logo" className={logo} />
+            </Grid>
+            <Grid item>
+              <Box className={classes.authWrapper}>
+                <Link to="" className={classes.sitterLink}>
+                  Become A Sitter
+                </Link>
+                <Link to="/login" className={classes.link}>
```
[the remark this answers is not in the document]
Comment (ababber) [reply client/src/components/AuthHeader/AuthHeader.tsx:27]: Thanks for the suggestion! That definitely makes the code more concise.

File: client/src/components/AuthHeader/useStyles.ts [hunk 7]
```diff
@@ -20,15 +31,27 @@ const useStyles = makeStyles((theme) => ({
...
-    marginRight: 35,
+    marginRight: 10,
+    display: 'flex',
+    alignItems: 'center',
+  },
+  signupBtn: {
+    width: 160,
+    height: 50,
+    borderRadius: theme.shape.borderRadius,
+    backgroundColor: '#F14140',
```
[the remark this answers is not in the document]
Comment (ababber) [reply client/src/components/AuthHeader/useStyles.ts:50]: Thanks for pointing this out! The color was part of the theme, but I forgot to clean up the useStyles file.

**Review by rajivtitus — APPROVED** [review]
LGTM! Great work implementing those changes! 💯

**Pull request merged**